src/hooks/LoggingHook.ts

```diff
@@ -1,30 +1,30 @@
-import type { FlagValue, Hook, HookContext, HookHints, ResolutionDetails } from '@openfeature/nodejs-sdk'
-import { EOL } from 'node:os'
+import type { Hook, HookContext, ResolutionDetails, FlagValue } from '@openfeature/js-sdk';
+import { EOL } from 'os';
 
 /**
- * A hook that adds standard Logging data.
+ * A hook that simply logs at every life-cycle stage.
  */
 export class LoggingHook implements Hook {
-  readonly name = 'logging'
+  name = 'logging';
 
-  constructor() {}
-
-  before(hookContext: HookContext, _hookHints: HookHints) {
-    console.log(`Running 'before' logger hook for flag: ${hookContext.flagKey}`)
-    console.log(JSON.stringify(hookContext.context, undefined, 2))
+  before(hookContext: HookContext) {
+    console.log(`Running 'before' logger hook for flag: ${hookContext.flagKey}`);
+    console.log(JSON.stringify(hookContext.context, undefined, 2));
   }
 
-  after(hookContext: HookContext, details: ResolutionDetails<FlagValue>, _hookHints: HookHints) {
-    console.log(`Running 'after' logger hook for flag: ${hookContext.flagKey}`)
-    console.log(`Evaluation details:${EOL}${JSON.stringify(details, undefined, 2)}`)
+  after(hookContext: HookContext, details: ResolutionDetails<FlagValue>) {
+    console.log(`Running 'after' logger hook for flag: ${hookContext.flagKey}`);
+    console.log(`Evaluation details:${EOL}${JSON.stringify(details, undefined, 2)}`);
   }
 
-  finally(hookContext: HookContext, _hookHints: HookHints) {
-    console.log(`Running 'finally' logger hook for flag: ${hookContext.flagKey}`)
+  finally(hookContext: HookContext) {
+    console.log(
+      `Running 'finally' logger hook for flag: ${hookContext.flagKey}`
+    );
   }
 
-  error(hookContext: HookContext, error: Error, _hookHints: HookHints) {
-    console.log(`Running 'error' logger hook for flag: ${hookContext.flagKey}`)
-    console.error(error)
+  error(hookContext: HookContext, err: Error) {
+    console.log(`Running 'error' logger hook for flag: ${hookContext.flagKey}`);
```

Comment:
trailing semicolons have somehow appeared in this file

Reply:
Reran Prettier should be resolved now